examples/gen_test_data/README.md

```diff
@@ -0,0 +1 @@
+Refering to [generate-test-data.md](..\docs\how-to-guides\generate-test-data.md) to get more details about `gen_test_data` package.
```

Comment:
# Generate test data

Please kindly see [this doc]() for detailed steps of how to generate test data.